### PR TITLE
Add all positive cases for TomEE

### PR DESCRIPTION
**Pull request by jannahsherif**

### CODE REVIEW

**Comment by m-reza-rahman** [comment]
This looks a lot better. I’ll review closely as soon as I can.

**Comment by jannahsherif** [comment]
Thank you !

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -143,6 +143,11 @@ jobs:
           mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=web -DjavaVersion=8 -Druntime=tomee -Ddocker=yes -DoutputDirectory=app/tomee -Dgoals="clean package"
           rm -rf app/tomee
 
+      - name: Run Archetype for EE 8 Web Profile, SE 8, TomEE, with Docker
```
Comment (m-reza-rahman) [inline .github/workflows/nightly.yml:146]: This is a duplicate of the one above. Please fix the one above.

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -330,11 +335,26 @@ jobs:
...
+      - name: Run Archetype for EE 8 Web Profile, SE 11, TomEE, with Docker
+        run: |
+          mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=web -DjavaVersion=11 -Druntime="tomee" -Ddocker=yes -DoutputDirectory="app/tomee" -Dgoals="clean package"
+          rm -rf app/tomee
 
       - name: Run Archetype for EE 9.1 Web Profile, SE 11, TomEE
         run: |
           mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=9.1 -Dprofile=web -DjavaVersion=11 -Druntime="tomee" -Ddocker=yes -DoutputDirectory="app/tomee" -Dgoals="clean package"
           rm -rf app/tomee
+
```
Comment (m-reza-rahman) [inline .github/workflows/nightly.yml:353]: This is a duplicate of the one above. Please fix the one above.

**Review by m-reza-rahman — CHANGES_REQUESTED** [review]
Please address review comments.

**Comment by jannahsherif** [comment]
Thank you for the feedback, but i am not sure i understand where the duplicate is? One is with docker and one is without docker support

**Comment by m-reza-rahman** [comment]
The duplicates are the commands themselves, not the labels. They are all asking for a Dockerfile to be generated.
  

 From: jannahsherif ***@***.***>Sent: Tuesday, July 9, 2024 8:32 PMTo: eclipse-ee4j/starter ***@***.***>Cc: Reza Rahman ***@***.***>; Comment ***@***.***>Subject: Re: [eclipse-ee4j/starter] Add all positive cases for TomEE (PR #312) 
Thank you for the feedback, but i am not sure i understand where the duplicate is? One is with docker and one is without docker support

**Pull request merged**